src/api-binder/report.ts

```diff
@@ -141,15 +142,30 @@ export async function startReporting() {
 	);
 	const doReport = async () => {
 		if (!reportPending) {
-			throttledReport(reportConfigs);
+			return throttledReport(reportConfigs);
 		}
 	};
 
 	// If the state changes, report it
 	deviceState.on('change', doReport);
```

Comment:
there's no more setInterval so unsure if this comment is still required. The throttle is to prevent device state changes and the interval calling report too quickly and the interval is to call report when no device state events are emitted.

Am I missing something ?

Reply:
Discussed further in fd/meet

Reply:
the delay and throttle are the same value so effectively the deviceState event listener is useless